### PR TITLE
Customer requisition toolbar disappears

### LINKED ISSUE #1707: Requisition details not visible
## What went wrong? 😲

When refreshing a page on request requisition page (could be same for internal order), details are hidden

## Expected behaviour 🤔

Details are not hidden

## How to Reproduce 🔨

https://user-images.githubusercontent.com/26194949/236709421-197709ac-056d-40f8-bc82-b155679ad0ce.mov

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #1707

# 👩🏻‍💻 What does this PR do? 
 
I'm putting this one down to 'it works now' and walking away. I spent a long time trying to find why the react portal isn't behaving and then updated the translation namespace in the footer component and now the portal works. The namespace change was just an offroad, I noticed that it was pulling in `replenishment` but is part of `distribution`. No functional change, it all works as it did.. 🤷 

The other changes here are offroads

# 🧪 How has/should this change been tested? 

Navigate to a response requisition, refresh page.

## 📃 Documentation

_No user facing changes_

### CODE REVIEW

File: client/packages/common/src/hooks/useDrawer/useDrawer.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import create from 'zustand';
+import { create } from 'zustand';
```
Comment (mark-prins) [inline client/packages/common/src/hooks/useDrawer/useDrawer.ts:1]: change required in the zustand update from ages ago. was throwing a console warning.

File: client/packages/requisitions/src/ResponseRequisition/DetailView/Footer/Footer.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ export const Footer: FC = () => {
   return (
     <AppFooterPortal
       Content={
-        data ? (
+        data && (
```
Comment (mark-prins) [inline client/packages/requisitions/src/ResponseRequisition/DetailView/Footer/Footer.tsx:32]: no functional change - just making it consistent with the request requisition version

File: client/packages/requisitions/src/ResponseRequisition/DetailView/Footer/StatusChangeButton.tsx [hunk 3]
```diff
@@ -60,7 +60,7 @@ const getButtonLabel =
 const useStatusChangeButton = () => {
   const { status, update } = useResponse.document.fields('status');
   const { success, error } = useNotification();
-  const t = useTranslation('replenishment');
+  const t = useTranslation('distribution');
```
Comment (mark-prins) [inline client/packages/requisitions/src/ResponseRequisition/DetailView/Footer/StatusChangeButton.tsx:63]: the fix 🤷‍♂️

**Review by roxy-dao — APPROVED** [review]
LGTM ~ 👀

**Pull request merged**